src/WebJobs.Script/EnvironmentSettingNames.cs

```diff
@@ -23,5 +21,6 @@ public static class EnvironmentSettingNames
         public const string AzureWebJobsExtensionsPath = "AzureWebJobs_ExtensionsPath";
         public const string AzureWebsiteAppCountersName = "WEBSITE_COUNTERS_APP";
         public const string AzureWebJobsSecretStorageType = "AzureWebJobsSecretStorageType";
+        public const string AppInsightsInstrumentationKey = "APPINSIGHTS_INSTRUMENTATIONKEY";
```

Comment:
The portal writes it -- it was chosen b/c App Service already had support for writing this key so we can piggy-back on some of that.